restapi/api.go

```diff
@@ -21,10 +25,31 @@ func Router(db *gorm.DB) (http.Handler, error) {
 	}
 
 	rest.GetHandler(r, "/health", api.health)
-
+	rest.PostHandler(r, "/authors", api.authorCreate)
 	return r, nil
 }
 
+func (api api) authorCreate(w http.ResponseWriter, r *http.Request) render.Renderer {
```

Comment:
naming nitpick but make we should name it `CreateAuthor` this way what we are doing comes first